.github/actions/create-github-release-action/action.yml

```diff
@@ -0,0 +1,25 @@
+---
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2025 The Linux Foundation
+
+name: "Create Github release"
+
+inputs:
+  VERSION:
+    description: "Version to release on Github"
+    type: string
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: "Create Github release"
+      shell: bash
+      run: |
+        if gh release create "${{ inputs.VERSION }}" --generate-notes; then
```

Comment:
Have you tested the `gh` command? I am asking this because when we tried to use it in the `omec-project` it did not work, and we had to move back to use the API instead.

Reply:
Yes, I've tested this on a non-production repo. Could you point me to where this failed on omec-project?

Reply:
I think it was in this PR: https://github.com/omec-project/.github/pull/1 from Gulsum

Reply:
Sorry, I should be more specific. Could you point me to a workflow run where the old code ran and failed?

Reply:
My bad.... my comment was incomplete. I think it was for the `tag-release` action but I am not certain. Let me check with Gulsum through Slack

Reply:
I've checked a bunch of repos, and can't find an instance of the job failing due to the `gh` command. Unless there's a specific run that can be pointed to, I'd like to merge this as-is, so that I can at least take a crack at troubleshooting the issue myself. Again, I've tested that this command works in the same way as the REST request, and it is significantly simpler and less error-prone. So if we see some initial failures, I'd like a chance to look at what's happening and do some troubleshooting.